starter-code/pizza.js

```diff
@@ -1,85 +1,153 @@
-  greenPeppers: {name: 'Green Peppers', price: 1},
-  whiteSauce: {name: 'White sauce', price: 3},
-  glutenFreeCrust: {name: 'Gluten-free crust', price: 5}
-}
+  pepperonni: { name: "Pepperonni", price: 1 },
+  mushrooms: { name: "Mushrooms", price: 1 },
+  greenPeppers: { name: "Green Peppers", price: 1 },
+  whiteSauce: { name: "White sauce", price: 3 },
+  glutenFreeCrust: { name: "Gluten-free crust", price: 5 }
+};
 
 // Initial value of the state (the state values can change over time)
 var state = {
   pepperonni: true,
   mushrooms: true,
   greenPeppers: true,
-  whiteSauce: false,
-  glutenFreeCrust: false
-}
+  whiteSauce: true,
+  glutenFreeCrust: true
+};
 
 // This function takes care of rendering the pizza based on the state
 // This function is triggered once at the begining and everytime the state is changed
 function renderEverything() {
-  renderPepperonni()
-  renderMushrooms()
-  renderGreenPeppers()
-  renderWhiteSauce()
-  renderGlutenFreeCrust()
-
-  renderButtons()
-  renderPrice()
+  renderPepperonni();
+  renderMushrooms();
+  renderGreenPeppers();
+  renderWhiteSauce();
+  renderGlutenFreeCrust();
+
+  renderButtons();
+  renderPrice();
 }
 
 function renderPepperonni() {
-  document.querySelectorAll('.pep').forEach(function($pep){
+  document.querySelectorAll(".pep").forEach(function($pep) {
     if (state.pepperonni) {
       $pep.style.visibility = "visible";
-    }
-    else {
+    } else {
       $pep.style.visibility = "hidden";
     }
-  })
+  });
 }
 
 function renderMushrooms() {
   // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll(".mushroom").forEach(function(mush) {
+    if (state.mushrooms) {
+      mush.style.visibility = "visible";
+    } else {
+      mush.style.visibility = "hidden";
+    }
+  });
 }
 
 function renderGreenPeppers() {
   // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll(".green-pepper").forEach(function(greenpep) {
+    if (state.greenPeppers) {
+      greenpep.style.visibility = "visible";
+    } else {
+      greenpep.style.visibility = "hidden";
+    }
+  });
 }
 
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  document.querySelectorAll(".sauce").forEach(function(sauce) {
+    if (state.whiteSauce) {
+      sauce.classList.add("sauce-white");
+    } else {
+      sauce.classList.remove("sauce-white");
+    }
+  });
 }
 
 function renderGlutenFreeCrust() {
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  document.querySelectorAll(".crust").forEach(function(crust) {
+    if (state.glutenFreeCrust) {
+      crust.classList.add("crust-gluten-free");
+    } else {
+      crust.classList.remove("crust-gluten-free");
+    }
+  });
 }
 
 function renderButtons() {
   // Iteration 3: add/remove the class "active" of each `<button class="btn">`
```

Comment:
This works and is perfectly fine. If you want, have a look at Object.keys(). With this, you can reduce your code by a lot and make it cleaner. Let me know if you need more help with this :)